comixed-opds/src/main/java/org/comixedproject/opds/rest/OPDSListsController.java

```diff
@@ -56,7 +58,9 @@
    * @return the reading list links
    * @throws OPDSException if an error occurs
    */
-  @GetMapping(value = "/opds/lists", produces = MediaType.APPLICATION_XML_VALUE)
+  @GetMapping(
+      value = {"/opds/lists", "/opds/all"},
```

Comment:
Do we need two separate URLs that return the same content?

Reply:
Sorry it was a leftover from me doing tests against Kuboo. I've removed it